assets/activation-scripts/activate.d/attach-inplace.bash

```diff
@@ -1,29 +1,43 @@
+expiring_pid="$$"
+# Put a 2 second timeout on the activation
+"$_flox_activations" \
+  --runtime-dir "$FLOX_RUNTIME_DIR" \
+  attach \
```

Comment:
Yeah because for `eval "$(flox activate)"` there's time between when `flox activate` prints something and the shell actually eval'ing it, and we don't want the watchdog to clean up in that time